libs/counterfactuals/src/lib/CounterfactualList.tsx

```diff
@@ -336,7 +336,9 @@ export class CounterfactualList extends React.Component<
       this.props.data?.feature_names_including_target.indexOf(id);
     this.props.setCustomRowProperty(`Data${keyIndex}`, false, newValue);
     this.setState((prevState) => {
-      prevState.data[id] = toNumber(newValue);
+      prevState.data[id] = newValue?.endsWith(".")
+        ? newValue
```

Comment:
So if the value has "." then it will be treated as a string? Can we test the prediction what it returns in this case?

Reply:
yeah that's a good point. We will throw exception while predicting, as data must be int, float or bool:
![image](https://user-images.githubusercontent.com/91754176/171513807-4dff65a5-c830-4f0d-9af0-283796edb109.png)

Reply:
I am thinking of 2 approaches:
(1) From UI side, we check and convert to number if there is any string.
(2) Without updating UI, we create this.state.displayedData, where diaplayedData can be string | number, and we limit this.state.data to be number[] only.

Reply:
Added this.state.displayedData. So this.state.data is the same as before, the prediction call won't hit error in this case. 
And we allow users to input decimal point by adding this.state.displayedData

Reply:
No, this is only for UI display. The request we send to /predict call is still decimal value.